packages/core/frontend/sirius-components-core/src/workbench/Site.tsx

```diff
@@ -157,11 +68,5 @@ export const Site = ({ editingContextId, readOnly, side, expanded, toggleExpansi
     );
   }
 
-  let classSite = side === 'left' ? classes.leftSite : classes.rightSite;
-  return (
-    <div className={classSite} data-testid={`site-${side}`}>
-      {viewSelector}
-      {selectedView}
-    </div>
-  );
+  return null;
```

Comment:
The fact that a site can return null is odd, wouldn't that be a part instead of a site?